src/components/Send/SendScreen.js

```diff
@@ -264,8 +264,13 @@ const getTransactionData = async (
   return await walletManager.createUnsignedTx(utxos, address, sendTokenList, defaultTokenEntry, serverTime)
 }
 
-const recomputeAll = async ({amount, address, utxos, sendAll, defaultAsset, selectedTokenMeta, tokenBalance}) => {
+const recomputeAll = async ({amount, addressInput, utxos, sendAll, defaultAsset, selectedTokenMeta, tokenBalance}) => {
   let amountErrors = validateAmount(amount, selectedTokenMeta)
+  address = addressInput;
+  if (addressInput !== undefined && addressInput.split('.').length > 1) {
+      address = await getUnstoppableDomainAddress(addressInput);
```

Comment:
how long does this take?
does this add lag to the input?
what happens if it fails?

Reply:
In me testing the domains usually resolved in <0.5s, in the iOS simulator it was instant for me.
Sometimes when stressing the application, by adding and removing the last letter in `brad.crypto`, it would take slightly longer however it wasn't long enough that I feel it would affect user experience.

I check for `.` such that if a user enters a regular address, and not a domain, the code will not run at all.

If a user enters a domain name that is not registered, or a domain that does not have a Cardano address bound to it, the  normal `addressErrors.invalidAddress` is thrown and 'Please enter a valid address' is shown (2nd image above).

Reply:
what happens if the user loses internet connection add they are typing?

Reply:
I tested without internet access and it seemed that `revalidate` does not succeed when the device is offline. For the user this manifests as the application showing 'Invalid address' regardless to whether the domain is valid or not. This does not slow typing either and runs just as it did prior to the addition.

Once internet is restored the resolution is instant again and the domain address shows as valid.